### ISSUE TITLE
What will discover do？

### ISSUE THREAD

**Issue by i6448038** [phpipam/phpipam-agent#40]
always pring "Didn't find any address to scan"

**Comment by KenK73** [comment]
I am seeing the same, but it is inconsistent.  I have many subnets in different geographic locations. I have 4 separate scan agents setup for each area.  All of these scan agents can successfully perform the update function, but only 2 can perform the discover function.

When it works, and you have "Discover new hosts" enabled on a subnet, it will find IP's not yet registered, and add them to your database with a little "auto-discovered on this date" note.

That said, there appears to be a bug or some db corruption preventing some of my scanners from working.

I am running IPAM v1.3.2 and I am using both the pierrecdn/phpipam and pierrecdn/ipam-agent Docker containers on top of Photon OS 2.0.

**Comment by KenK73** [comment]
Two different hosts (same Docker pulls). Both "update" fine, only 1 will "discover":

root@1b83c85d77e5:/opt/phpipam-agent#  /usr/local/bin/php /opt/phpipam-agent/index.php discover
--------------------
Error: Didn't find any address to scan!
--------------------
root@1b83c85d77e5:/opt/phpipam-agent#


root@29a6b3cf61fe:/opt/phpipam-agent# /usr/local/bin/php /opt/phpipam-agent/index.php discover

13.68.12.1   : xmt/rcv/%loss = 1/1/0%, min/avg/max = 14.1/14.1/14.1
13.68.12.2   : xmt/rcv/%loss = 1/1/0%, min/avg/max = 0.51/0.51/0.51
13.68.12.3   : xmt/rcv/%loss = 1/1/0%, min/avg/max = 0.65/0.65/0.65
13.68.12.4   : xmt/rcv/%loss = 1/1/0%, min/avg/max = 0.61/0.61/0.61
13.68.12.126 : xmt/rcv/%loss = 1/0/100%
root@29a6b3cf61fe:/opt/phpipam-agent#

**Comment by nike-on** [comment]
Any news about that problem? ;)
We have 2024 now, and problem still occur :(
